controllers/config/operator_constants.go

```diff
@@ -11,6 +11,9 @@ const (
 	GrafanaPluginsPath      = "/var/lib/grafana/plugins"
 	GrafanaProvisioningPath = "/etc/grafana/provisioning/"
 
+	// Grafonnet
+	GrafonnetEmbedPath = "./embeds/grafonnet/grafonnet-lib/"
```

Comment:
Would be nice to add a note somewhere that would explain where to get the library from, which files to copy, and so on. - Just to make our life easier in an event when we need to update the library.

Reply:
I added readme in the embeds dir. I also switched to using a gitmodule, which means updating the lib is just `git submodule update` no longer any need to manually move things around. 

ps. I've edited the .gitignore to hopefully prevent overbloating the grafonnet-lib directory